### DISCUSSION TITLE
Year selection buttons get cut off when navigating around profile

### DISCUSSION THREAD

**Discussion by danielleos** [community/community#30938] [Other Feature Feedback, Questions, & Ideas] [answered]
Noticed some weird behaviour when I was navigating around my profile. The year buttons in the contribution section get cut off when coming back to the 'Overview' tab after viewing a different tab (i.e. Repos, Projects, Packages, Stars).

OS & Browser: macOS (12.4), Chrome (Version 104.0.5112.101)

Steps to recreate:
1. Go to your profile overview (scroll to the contributions section)
    - The year selection buttons should display the full year
    ![Screenshot 2022-08-26 at 16 10 31](https://user-images.githubusercontent.com/16842000/186936532-7ebe09ef-2dcb-4f2b-b94f-aa92cb6a09f9.png)
2. Navigate to a different tab in your profile (e.g. Repositories, Projects)
    - Scroll to the contributions section
    - The year buttons are cut off so you can't see the full year (resolved with a refresh of the page)
    ![Screenshot 2022-08-26 at 16 10 46](https://user-images.githubusercontent.com/16842000/186936543-d1efe690-f0e3-4d43-97d9-cca5ab74e375.png)

**Comment by melvincwng** [comment, marked as answer]
@danielleos Hi Danielle 👋, thanks for flagging this bug to the community.

I have checked this issue on my end, and indeed I was able to replicate this bug as well.

My suggestion would be for you to raise a ticket to the GitHub Support team over [here](https://support.github.com/contact/bug-report). 

Explain the issue at hand, provide link to this discussion, and kindly provide them several days to check and resolve this issue.

Hope this answers your question  😊

**Reply by danielleos** [reply]
Thanks @melvincwng will raise a ticket!

**Comment by danielleos** [comment]
Response from GitHub:

> Thanks for reaching out to GitHub Support to report problems when navigating back to the Overview tab on a users profile. I do see this same problem in my testing. I have taken care of forwarding your report to our team for investigation. We will follow-up with you once this has been fixed.

**Comment by danielleos** [comment]
A fix has been deployed - confirmed working on my end
